tests/layer_tests/common/layer_test_class.py

```diff
@@ -10,7 +10,7 @@
 
 import numpy as np
 from common.constants import test_device, test_precision
-from common.layer_utils import IEInfer, InferAPI20
+from common.layer_utils import InferAPI20
```

Comment:
should we rename `InferAPI20` => `InferAPI`, because we don't need to distinguish between 1.0 and 2.0 anymore

Reply:
ok